### ISSUE TITLE
ErrCreateFailed always return `internal error`

### ISSUE THREAD

**Issue by toml01** [CosmWasm/wasmd#240]
https://github.com/CosmWasm/wasmd/blob/master/x/wasm/internal/types/errors.go#L12

Seems like Cosmos SDK overrides any error with code 1 with `internal error`.
https://github.com/cosmos/cosmos-sdk/issues/6916

A quick fix would be to just change the code for `ErrCreateFailed` I guess.

**Comment by toml01** [comment]
https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/errors.md#registration
> Must be greater than one, as a code value of one is reserved for internal errors.

**Comment by ethanfrey** [comment]
Yes, let us bump all the error codes by one in the next (post 0.10) release
